submodules.json

```diff
@@ -8,5 +8,10 @@
         "name" : "node-boilerplate",
         "url" : "https://github.com/hagopj13/node-express-boilerplate",
         "tags" : ["node", "nodejs"]
+    },
+     {
```

Comment:
Please lint the JSON properly 😅. Would be great if you could remove that whitespace.